Jenkinsfile

```diff
@@ -524,14 +577,24 @@ pipeline {
                     " -TestPlatform 'Azure'" +
                     " ${env.LISAV2_PARAMS} " +
                     " -OsVHD '${env.CapturedVHD}'" +
-                    " -XMLSecretFile '${env.Azure_Secrets_File}'"
+                    " -XMLSecretFile '${env.Azure_Secrets_File}'" +
+                    " -ExitWithZero"
                 )
             }
           }
           post {
             always {
               junit "Report\\*-junit.xml"
               archiveArtifacts "TestResults\\**\\*"
+              script {
+                  def passCount = GetTestResults("pass")
+                  def abortCount = GetTestResults("abort")
+                  def failCount = GetTestResults("fail")
+                  def skippedCount = GetTestResults("skipped")
+                  FUNC_PASS_ONAZURE = FUNC_PASS_ONAZURE.toInteger() + passCount.toInteger()
+                  FUNC_ABORT_ONAZURE = FUNC_ABORT_ONAZURE.toInteger() + abortCount.toInteger()
+                  FUNC_FAIL_ONAZURE = FUNC_FAIL_ONAZURE.toInteger() + failCount.toInteger()
```

Comment:
I don't see the Skip ones here, we don't want/need to have that in the DB or is it because the jenkins json does not know about that label? @LiliDeng 
Otherwise I'm good, we can merge this ASAP.

Reply:
Thanks for review, I also consider this question.
We already have count for all, pass, abort, then skipped will be (all - pass - abort), this is my thoughts, if need added, please let know.

Reply:
that would be largely confusing, the totals will simply not add up.
And it would be not transparent on where are the "missing" tests.

Please merge this at your own responsibility if you want to keep it like this.

You'll have:
Total: 90
Passed: 60
Failed: 8
Aborted: 12
(missing 10 tests from the total - the skipped ones).

Reply:
I will add skipped columns in the database, then update Jenkins file accordingly.